lib/src/page_route.dart

```diff
@@ -43,12 +44,14 @@ class SwipeablePageRoute<T> extends CupertinoPageRoute<T> {
   /// If set to `false`, the user can start dragging anywhere on the screen.
   final bool canOnlySwipeFromEdge;
 
+  /// If [canOnlySwipeFromEdge] is set to `true`, this value controls 
+  /// width of gesture detection area
+  final double backGestureDetectionWidth;
+
   /// If [canOnlySwipeFromEdge] is set to `true`, this value controls how far
   /// away from the left (LTR) or right (RTL) screen edge a gesture must start
   /// to be recognized for back navigation.
-  ///
-  /// In [CupertinoPageRoute], this value is `20`.
-  final double backGestureDetectionWidth;
+  final double backGestureDetectionStart;
```

Comment:
What do you think about renaming this to `backGestureDetectionStartOffset`? That might be more explicit and easier to understand without having to read the doc comment.

Reply:
backGestureDetectionStart looks fine

Reply:
Sorry, I mean backGestureDetectionStartOffset
You should fix change